tests/integrationv2/providers.py

```diff
@@ -323,8 +323,6 @@ def setup_client(self):
         # Additional debugging that will be captured incase of failure
         cmd_line.extend(['-debug', '-tlsextdebug'])
 
-        if self.options.cert is not None:
```

Comment:
What happened here? Was this not used or not working before?

Reply:
We don't need this line. We already have a option for client auth certificates, which is the only time in our tests where Openssl requires a certificate. Conversely, the jdk client does require certificates to authenticate the server. So when I add this line: https://github.com/awslabs/s2n/blob/ae574ad00903980e2929a817ed427919dfc8679c/tests/integrationv2/test_happy_path.py#L31, Openssl will error. The alternative to this is to get the java client to trust everything so that it doesn't require a certificate from the server.

Reply:
👍